fe/fe-core/src/main/java/com/starrocks/server/GlobalStateMgr.java

```diff
@@ -1790,6 +1791,20 @@ public long saveHeader(DataOutputStream dos, long replayedJournalId, long checks
         return checksum;
     }
 
+    // TODO [meta-format-change]
```

Comment:
meaning this pr need to go to 3.0 first?

Reply:
methods marked with [meta-format-change] will be used in 3.1.